packages/@stylexjs/postcss-plugin/README.md

```diff
@@ -135,3 +135,12 @@ useCSSLayers: boolean; // Default: false
 
 Enabling this option switches Stylex from using `:not(#\#)` to using `@layers`
 for handling CSS specificity.
+
+### importSources
```

Comment:
We should add this to the `docs` package's page for the postcss-plugin

Reply:
I've added it to the docs page in this comment.
https://github.com/facebook/stylex/pull/1063/commits/c1c6a10cc0c37e1d9031832157450a070944a3ec